rest_framework/routers.py

```diff
@@ -26,7 +26,8 @@
 from rest_framework.compat import NoReverseMatch
 from rest_framework.response import Response
 from rest_framework.reverse import reverse
-from rest_framework.schemas import SchemaGenerator, SchemaView
+from rest_framework.schemas import SchemaGenerator
+from rest_framework.schemas.views import SchemaView
```

Comment:
This is the only point where we can't keep the external API of `schemas` the same. 

```
┌────────────┐       ┌────────┐
│  schemas   │──────▶│ views  │
└────────────┘       └────────┘
       │                  │
       │                  │
       │                  ▼
       │          ┌──────────────┐
       └─────────▶│schemas.views │
                  └──────────────┘
```

Only option would be to move `SchemaView` out of `schemas` entirely.

In basic cases `SchemaView` is not directly employed by the end user anyway.

Reply:
Yeah. Not a blocker, so long as the change is documented clearly in the release notes.